nexus/db-model/src/rack.rs

```diff
@@ -28,6 +29,22 @@ impl Rack {
             rack_subnet: None,
         }
     }
+
+    pub fn get_subnet(&self) -> Result<Ipv6Network, api::external::Error> {
```

Comment:
super nit: I think we usually don't use `get_` prefixes (per https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter)

Reply:
Done in [8ad101f](https://github.com/oxidecomputer/omicron/pull/4504/commits/8ad101fff837f1373947e10b207367b34bf7932f)